src/main/app/forms/validation/validators/isValidYearFormat.ts

```diff
@@ -8,39 +8,35 @@ import {
 
 import { LocalDate } from 'forms/models/localDate'
 
+const numberOfDigitsInAYear = 4
```

Comment:
+const numberOfDigitsInAYear: number = 4 ?

Reply:
don't think it adds any value. you can see its a number because its right next to it